bootcamp_content/projects/breakout/exercises/smashing-blocks/config.json

```diff
@@ -9,7 +9,7 @@
     "timePerFrame": 0.01,
     "repeat_delay": 10,
     "max_total_loop_iterations": 2000,
-    "max_repeat_until_game_over_iterations": 600,
+    "max_repeat_until_game_over_iterations": 300,
```

Comment:
I think this was added to make people define their own game_over criteria, but it is not given in the exercise text, that `repeat until game over` is not the tool to use?

Reply:
Actually no. I've reverted this. It was because it was running so slowly on my machine when it hit 500 iterations during other testing I was doing. It just leaked into this PR 😢 

Thanks for flagging!